### ISSUE TITLE
command -v

### ISSUE THREAD

**Issue by codersteinersky** [koalaman/shellcheck#2504]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [ ] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [x] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

if ! which make ar ld nm >/dev/null; then
 err "ODB patching needs: \"make ar ld nm\" but some aren't in the PATH or aren't installed"
fi

```

#### Here's what shellcheck currently says:
```

```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by codersteinersky** [comment]
Sorry, I only got SC2230 reported locally with SC version (0.7.0) and see it's now an opt-in only.  This didn't get flagged in the current version on the site.  Further research on your great issues explanations site describes the exact problematic behavior I was seeing when trying to use "command -v" in lieu of "which" to ensure all needed commands exist.
